### ISSUE TITLE
v1.0.4 release missing

### ISSUE THREAD

**Issue by dbuonomo** [concrete5-community-store/community_store_stripe#10]
Just noticed that v1.0.4 is no longer available as a release. It's the version we are currently using, so just wondering if that is an oversight or if there's a specific reason.

Thanks

**Comment by Mesuva** [comment]
I don't believe I deleted any releases, I may have just skipped creating a few versions.
So there wasn't a specific reason

**Comment by dbuonomo** [comment]
Ok, I think I know what happened. I did a git pull on the repo which must have been at v1.0.4. I don't think v1.0.4 was ever posted as a release.

I thought I installed it via a zip file download, but that is not the case.

Thanks
